include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.hpp

```diff
@@ -250,6 +250,30 @@ class Orbit : public Trajectory
         const Angle& anArgumentOfLatitude = Angle::Zero()
     );
 
+    /// @brief Constructs a Frozen orbit
```

Comment:
Just like the COE one can you update the docstring here enumerating the various cases 🙏